pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/util/HelixSetupUtils.java

```diff
@@ -53,126 +56,214 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static org.apache.pinot.common.utils.CommonConstants.Helix.LEAD_CONTROLLER_RESOURCE_NAME;
+
 
 /**
  * HelixSetupUtils handles how to create or get a helixCluster in controller.
  *
  *
  */
 public class HelixSetupUtils {
-
   private static final Logger LOGGER = LoggerFactory.getLogger(HelixSetupUtils.class);
 
   public static synchronized HelixManager setup(String helixClusterName, String zkPath,
-      String pinotControllerInstanceId, boolean isUpdateStateModel, boolean enableBatchMessageMode) {
-
+      String helixControllerInstanceId) {
     try {
```

Comment:
No need for try-catch. The caller is responsible for logging the exception

Reply:
Removed.